retroshare-gui/src/gui/settings/AppearancePage.cpp

```diff
@@ -142,6 +143,7 @@ bool AppearancePage::save(QString &errmsg)
 	Settings->setValueToGroup("StatusBar", "HideSound", QVariant(ui.checkBoxHideSoundStatus->isChecked()));
 	Settings->setValueToGroup("StatusBar", "HideToaster", QVariant(ui.checkBoxHideToasterDisable->isChecked()));
 	Settings->setValueToGroup("StatusBar", "ShowSysTrayOnStatusBar", QVariant(ui.checkBoxShowSystrayOnStatus->isChecked()));
+	Settings->setValueToGroup("StatusBar", "DisableSysTrayToolTip", QVariant(ui.checkBoxDisableSysTrayToolTip->isChecked()));
```

Comment:
I think pMainWindow->toggleStatusToolTip(ui.checkBoxDisableSysTrayToolTip->isChecked()); should be called here, otherwise the status could have been overwritten by MainWindow::updateStatus() in the meantime